components/layout/header.vue

```diff
@@ -1,13 +1,38 @@
 <template>
+  <!-- <div class="flex justify-between items-center p-10 shadow">
+    <div>
+      <div class="justify-start">
+        <NuxtLink to="/">
+          <BrandLogo class="w-24 h-24" />
+        </NuxtLink>
+      </div>
+    </div>
+    <div class="flex justify-center">
+      <span
+        class="text-gray-900 inline-flex items-center px-1 pt-1 font-medium"
+      >
+        Hotline 24/7
+      </span>
+    </div>
+
+    <MenuIcon class="h-6 w-8" />
+      
+    
+  </div> -->
   <Popover class="relative bg-white">
     <div class="max-w-7xl mx-auto px-4 sm:px-6">
       <div
         class="flex justify-between items-center border-b-2 border-gray-100 py-6 md:justify-start md:space-x-10"
       >
         <div class="flex justify-start lg:w-0 lg:flex-1">
-          <NuxtLink to="/">
-            <BrandLogo class="w-24 h-24" />
-          </NuxtLink>
+          <a href="#">
```

Comment:
Here you have a logo already. Revert that change.

Reply:
Should I remove the wehelp logo? Am I interpreting it right?